src/simulator/SimulatorModel.ts

```diff
@@ -6,6 +6,7 @@ export interface SimulatorState {
 
 export interface Scenario {
     totalPopulation: number;
+    initialNumDead: number,
```

Comment:
Starting scenario is used to set day 0 of the model instead of the runup period history, so the current number of dead, infected, costs etc are set in scenario now